tokio-trace/src/dispatcher.rs

```diff
@@ -52,7 +54,11 @@ impl Dispatch {
     where
         S: Subscriber + Send + Sync + 'static,
     {
-        Dispatch(Arc::new(subscriber))
+        static GEN: AtomicUsize = ATOMIC_USIZE_INIT;
```

Comment:
In this case, I don't think we're likely to have so many dispatchers that this counter is likely to overflow, even on a 32-bit machine.